### PR TITLE
Change in response

### PR DESCRIPTION
**Pull request by KhushalPShah**
Changed the Response formats for : 
- DB Collection Rules
- DB Configs
- DB Prepared Query

### CODE REVIEW

File: space-cli/cmd/modules/database/getters.go [hunk 1]
```diff
@@ -22,22 +21,23 @@ func GetDbRule(project, commandName string, params map[string]string) ([]*model.
...
-			}
-			objs = append(objs, s)
+		table := obj["table"].(string)
+		dbAlias := obj["dbAlias"].(string)
+		if table == "event_logs" || table == "invocation_logs" {
+			continue
 		}
+		meta := map[string]string{"project": project, "col": table, "dbAlias": dbAlias}
+
+		delete(obj, "schema")
```
Comment (sharadregoti) [inline space-cli/cmd/modules/database/getters.go:31]: I think this is not required, as the dbRule object in gateway doesn't contain schema field. So this line of code is not useful

File: space-cli/cmd/modules/database/getters.go [hunk 2]
```diff
@@ -53,21 +53,22 @@ func GetDbConfig(project, commandName string, params map[string]string) ([]*mode
...
-			// Generating the object
-			s, err := utils.CreateSpecObject("/v1/config/projects/{project}/database/{dbAlias}/config/{id}", commandName, meta, value)
-			if err != nil {
-				return nil, err
-			}
-			objs = append(objs, s)
+		obj := item.(map[string]interface{})
+		dbAlias := obj["dbAlias"]
+		configID := fmt.Sprintf("%s-config", dbAlias.(string))
+		meta := map[string]string{"project": project, "dbAlias": dbAlias.(string), "id": configID}
```
Comment (sharadregoti) [inline space-cli/cmd/modules/database/getters.go:59]:
```suggestion
		dbAlias := obj["dbAlias"].(string)
		configID := fmt.Sprintf("%s-config", dbAlias)
		meta := map[string]string{"project": project, "dbAlias": dbAlias, "id": configID}
```

**Review by sharadregoti — CHANGES_REQUESTED** [review]

File: space-cli/cmd/modules/database/getters.go (hunk 1 above)
Comment (KhushalPShah) [reply space-cli/cmd/modules/database/getters.go:31]: Yes
But @YourTechBud said that it was there previously, and hence do not remove it as it wont create any difference.

**Comment by sharadregoti** [comment]
@KhushalPShah In the config.go file. Change json annotation from "table" to "col" in databaseRule & databaseSchema struct

File: gateway/config/config.go [hunk 3]
```diff
@@ -112,14 +112,14 @@ type DatabaseConfig struct {
 
 // DatabaseSchema stores information of db schemas
 type DatabaseSchema struct {
-	Table   string `json:"table,omitempty" yaml:"table"`
+	Table   string `json:"col,omitempty" yaml:"table"`
```
Comment (YourTechBud) [inline gateway/config/config.go:115]:
```suggestion
	Table   string `json:"col,omitempty" yaml:"col"`
```

File: gateway/config/config.go [hunk 4]
```diff
@@ -112,14 +112,14 @@ type DatabaseConfig struct {
...
 	DbAlias string `json:"dbAlias,omitempty" yaml:"dbAlias"`
 	Schema  string `json:"schema,omitempty" yaml:"schema"`
 }
 
 // DatabaseRule stores information of db rule
 type DatabaseRule struct {
-	Table             string           `json:"table,omitempty" yaml:"table"`
+	Table             string           `json:"col,omitempty" yaml:"table"`
```
Comment (YourTechBud) [inline gateway/config/config.go:122]:
```suggestion
	Table             string           `json:"col,omitempty" yaml:"col"`
```

**Review by YourTechBud — CHANGES_REQUESTED** [review]

**Review by YourTechBud — APPROVED** [review]

**Review by sharadregoti — APPROVED** [review]

**Pull request merged**